plottable.d.ts

```diff
@@ -277,6 +277,19 @@ declare module Plottable {
 
 
 declare module Plottable {
+    module Utils {
+        class CallbackSet<CB extends Function> {
+            constructor();
+            add(value: CB): CallbackSet<CB>;
+            remove(value: CB): CallbackSet<CB>;
+            values(): CB[];
+            callCallbacks(...args: any[]): CallbackSet<CB>;
```

Comment:
Well `has` or `contains` is mainly for checking if the callback is there.  If it doesn't need to exist then sure we don't need it.